modules/web/src/app/wizard/step/cluster/template.html

```diff
@@ -544,6 +544,11 @@ <h4>IPv6</h4>
                  matTooltip="Enable to deploy User SSH Key Agent to the cluster. It cannot be changed once the cluster is created."></i>
             </mat-checkbox>
 
+            <mat-checkbox *ngIf="provider === NodeProvider.OPENSTACK"
+                          [formControlName]="Controls.RouterReconciliation">
+              Skip router reconciliation
```

Comment:
Please also add a tooltip. (Same comment for edit cluster dialog)
```suggestion
              Skip Router Reconciliation
```